inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/function/EncryptFunction.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.function;
+
+import java.security.SecureRandom;
+import javax.crypto.Cipher;
+import javax.crypto.KeyGenerator;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import lombok.Getter;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.flink.table.functions.ScalarFunction;
+
+/**
+ * EncryptFunction class. It is a custom function, used to encrypt the value in the string.
+ */
+public class EncryptFunction extends ScalarFunction {
+
+    private static final long serialVersionUID = -7185622027483662395L;
+
+    private static final String ENCODING = "UTF-8";
+
+    public static final String KEY_ALGORITHM_AES = "AES";
+
+    public static final String KEY_ALGORITHM_3DES = "DESede";
+
+    private static final String CIPHER_ALGORITHM_3DES = "DESede/ECB/PKCS5Padding";
+
+    public static final String SIGN_ALGORITHMS = "SHA1PRNG";
```

Comment:
would prefer to use private when the variable is not visit by outer class

Reply:
Thank you for your suggestion, which has been corrected.